packages/node_modules/nav-frontend-tabs/md/Tabs.overview.mdx

```diff
@@ -1,33 +1,27 @@
 import Example from './../../../../guideline-app/app/ui/components/example/Example';
-import Tabs from './../';
+import Tabs, { Tab } from './../';
 import Panel from 'nav-frontend-paneler';
 import Alertstripe from 'nav-frontend-alertstriper';
 
 ## Normal
 
 <Example>
-    <Tabs 
-        tabs={[
-            {"label": "Første"},
-            {"label": "Andre"},
-            {"label": "Tredje"}
-        ]}
-        onChange={() => {}}
-    />
+    <Tabs>
+        <Tab label='Første'>Content1</Tab>
+        <Tab label='Andre'>Content2</Tab>
+        <Tab label='Tredje'>Content3</Tab>
+    </Tabs>
 </Example>
 
 ```jsx
-<Tabs 
-    tabs={[
-        {"label": "Første"},
-        {"label": "Andre"},
-        {"label": "Tredje"}
-    ]}
-    onChange={() => {}}
-/>
+    <Tabs>
+        <Tab label='Første'>Content1</Tab>
+        <Tab label='Andre'>Content2</Tab>
+        <Tab label='Tredje'>Content3</Tab>
+    </Tabs>
 ```
 
-## Kombinasjon med panel
+## Manuelt med Panel
```

Comment:
Lurer på om ikke denne fortsatt bør hete "Kombinasjon med panel" og så heller vise til følgende bruksmønster:

```tsx
<Tabs>
    <Tab label='Første'>
        <Panel border>Content1</Panel>
    </Tab>
    <Tab label='Andre'>
        <Panel border>Content2</Panel>
    </Tab>
    <Tab label='Tredje'>
        <Panel border>Content3</Panel>
    </Tab>
</Tabs>
```

Da tenker jeg vi også bør bygge inn støtte for at komponenten automatisk stiler panelene riktig (uten `border-top` osv) som beskrevet i dagens infoboks. Da kan teksten i info-boksen også endres til å forklare dette.

Reply:
Siden vi vil tilby verktøy som er enkel og fleksibel, kanskje må vi undersøke litt mer om vi egentlig trenger å lage noen sånn.